motionplan/armplanning/constraint_handler.go

```diff
@@ -134,17 +134,6 @@ func newConstraintHandler(
 		handler.AddStateFSConstraint(name, constraint)
 	}
 
-	switch opt.MotionProfile {
-	case LinearMotionProfile:
-		constraints.AddLinearConstraint(motionplan.LinearConstraint{opt.LineTolerance, opt.OrientationTolerance})
-	case PseudolinearMotionProfile:
-		constraints.AddPseudolinearConstraint(motionplan.PseudolinearConstraint{opt.ToleranceFactor, opt.ToleranceFactor})
```

Comment:
This breaks pseudolinear constraints; those aren't in our API and MotionProfile is the only way to specify one.

Reply:
how is that different than `motionplan.PseudolinearConstraint`

Reply:
This line is what creates a `motionplan.PseudolinearConstraint`. The rest of the constraints can be created within `ConstraintsFromProtobuf` but pseudolinear isn't in protobuf.

Reply:
That sounds wonderful